### ISSUE TITLE
stumbled upon this while trying to setup authentication in React

### ISSUE THREAD

**Issue by CSTufts** [rishipr/mern-auth#6]
great scaffolding, I will probably use a little of it. 
Just letting you know you use classnames on your client react app, but its not included in the package.json

thanks!

**Comment by efendic** [comment]
Cheers for making this brother. Did you by any chance deployed this to heroku? I've been trying to do it almost whole working day today, no success. I followed the video provided, but no go. Last thing I got is "Error: ENOENT: no such file or directory, stat '/app/client/build/index.html'" and I get this after a successful build, which I get only when I leave out the heroku-postbuild script from the video.
